examples/labs/alex_controller/main.py

```diff
@@ -0,0 +1,12 @@
+from pitop import Pitop, alex_config
+from pitop.labs import RoverWebController
```

Comment:
In general, I think we should avoid the use of `Rover` and use `Robot` instead, as there are many different types of robots that aren't rovers. It's also consistent with `robotics kit` and the `robotics` module inside the SDK

Reply:
This isn't for generic robots, it's specifically for rover projects. Originally it was built so that Alex projects can be driven around without any boilerplate. Have a look at the `examples/labs/alex_controller` and `examples/labs/rover_controller`

Reply:
Ok, so you're meaning is that a "rover" is anything with two wheels and a camera? I still think we should generalize to robot instead of rover, Prax and Sam are both robots that could definitely use this controller and they aren't really rovers (which are generally things that drive on other planets :D)

Reply:
fair point yeah!